step-capstone/src/scripts/Categories.js

```diff
@@ -0,0 +1,80 @@
+const shopping = new Set(["bicycle_store",
```

Comment:
How did you arrive at this list of strings for each category? I'm guessing these wouldn't be a completely exhaustive list

Reply:
We used the list of supported trips https://developers.google.com/places/web-service/supported_types and grouped them into categories we felt reflected different preferences users have. It isnt exhaustive because there are some types (like accounting) that we do not want.